### DISCUSSION TITLE
Is it possible to adapt Figma UI Kit PrimeOne to custom theme?

### DISCUSSION THREAD

**Discussion by AgnesGehle** [primefaces/community#587] [Figma UI Kit] [answered]
Hey guys, 

we made our custom primeNG theme and took the "primeng-sass-theme" from github for that. Now we would like to use the Figma UI Kit for designing. In the description we saw that you took "Lara" light and dark as a base for that. So our question is if it's easily possible to adapt this to our needs and our own theme?

Best Regards
Agnes

**Comment by onursenture** [comment, marked as answer]
Hi,

Using the Tokens Studio plugin, you can easily create your custom design in Figma. However, we do not have a product connecting Figma and code directly. But, since the existing tokens use the variables in the primeng-sass-theme, your development team can transform the tokens you changed in Figma by checking the JSON outputs. We are also currently working on this.

Best regards,
Onur

**Comment by smartm0use** [comment]
Hello, any way to convert JSON coming from Tokens Studio plugin to theme.css for PrimeNg applications?

**Reply by onursenture** [reply]
There has yet to be an official support for this right now. We are working on it.

**Reply by janeznavteh100** [reply]
Any ETA for this function? 
Currenly tokens are not synced in token studio and sass variables. Very difficult to adapt. 
How others solve this issue?

**Reply by onursenture** [reply]
We cannot provide an ETA at this moment.

**Reply by GuillaumeWenabi** [reply]
Hey any news about that ? @onursenture

**Reply by onursenture** [reply]
In the initial phase, we implemented the theme designer in the PrimeVue (v4) and PrimeNG (v18) showcases. The Figma integration is still under development. You can explore the theme designer in the showcases by clicking the 'cog' icon in the top bar.

<img width="1276" alt="image" src="https://github.com/user-attachments/assets/18d8026d-aec4-49a2-87e4-518c252f8e63" />
